libs/thunk/mock/mock.h

```diff
@@ -7,8 +7,10 @@
 #include <io.h>
 
 extern std::function<decltype(_close)> close_handler;
-extern std::function<decltype(CloseHandle)> close_handle_handler;
 extern std::function<decltype(CreateFileW)> create_file_handler;
+extern std::function<decltype(CloseHandle)> close_handle_handler;
```

Comment:
I'd recommend reverting this move, keep them in alphabetical order (that goes for line 14 too)